Assets/MixedRealityToolkit/_Core/Definitions/Devices/ControllerMappingLibrary.cs

```diff
@@ -15,204 +14,220 @@ public static class ControllerMappingLibrary
-        /// INDEXFINGER_LEFT_CONTROLLER
+        /// Windows Mixed Reality Motion Controller: Right Touchpad Vertical Movement<para/>
+        /// Valve Knuckles Controller: Left Controller Index Finger Cap Sensor<para/>
         /// </summary>
-        public const string MIXEDREALITY_AXIS20 = "MIXEDREALITY_AXIS20";
+        public const string AXIS_20 = "AXIS_20";
 
         /// <summary>
-        /// INDEXFINGER_RIGHT_CONTROLLER
+        /// Valve Knuckles Controller: Right Controller Index Finger Cap Sensor<para/>
         /// </summary>
-        public const string MIXEDREALITY_AXIS21 = "MIXEDREALITY_AXIS21";
+        public const string AXIS_21 = "AXIS_21";
 
         /// <summary>
-        /// MIDDLEFINGER_LEFT_CONTROLLER
+        /// Valve Knuckles Controller: Left Controller Middle Finger Cap Sensor<para/>
         /// </summary>
-        public const string MIXEDREALITY_AXIS22 = "MIXEDREALITY_AXIS22";
+        public const string AXIS_22 = "AXIS_22";
 
         /// <summary>
-        /// MIDDLEFINGER_RIGHT_CONTROLLER
+        /// Valve Knuckles Controller: Right Controller Middle Finger Cap Sensor<para/>
         /// </summary>
-        public const string MIXEDREALITY_AXIS23 = "MIXEDREALITY_AXIS23";
+        public const string AXIS_23 = "AXIS_23";
 
         /// <summary>
-        /// RINGFINGER_LEFT_CONTROLLER
+        /// Valve Knuckles Controller: Left Controller Ring Finger Cap Sensor<para/>
         /// </summary>
-        public const string MIXEDREALITY_AXIS24 = "MIXEDREALITY_AXIS24";
+        public const string AXIS_24 = "AXIS_24";
 
         /// <summary>
-        /// RINGFINGER_RIGHT_CONTROLLER
+        /// Valve Knuckles Controller: Right Controller Ring Finger Cap Sensor<para/>
         /// </summary>
-        public const string MIXEDREALITY_AXIS25 = "MIXEDREALITY_AXIS25";
+        public const string AXIS_25 = "AXIS_25";
 
         /// <summary>
-        /// PINKYFINGER_LEFT_CONTROLLER
+        /// Valve Knuckles Controller: Left Controller Pinky Finger Cap Sensor<para/>
         /// </summary>
-        public const string MIXEDREALITY_AXIS26 = "MIXEDREALITY_AXIS26";
+        public const string AXIS_26 = "AXIS_26";
 
         /// <summary>
-        /// PINKYFINGER_RIGHT_CONTROLLER
+        /// Valve Knuckles Controller: Right Controller Pinky Finger Cap Sensor<para/>
         /// </summary>
-        public const string MIXEDREALITY_AXIS27 = "MIXEDREALITY_AXIS27";
+        public const string AXIS_27 = "AXIS_27";
 
         #endregion Constants
 
         #region InputAxisConfig
 
+#if UNITY_EDITOR
+
         /// <summary>
         /// Get the InputManagerAxis data needed to configure the Input Mappings for a controller
         /// </summary>
-        /// <param name="type">The type of controller to retrieve configuration for</param>
         /// <returns></returns>
-        public static InputManagerAxis[] GetOpenVRInputManagerAxes
+        public static InputManagerAxis[] UnityInputManagerAxes => new[]
```

Comment:
Willing to have multiple arrays (one per known controller type + a generic one)?
We can make a very readable UI by allowing users to select their controller type and then see the controls available for mapping. Ping me if you have questions on this feedback.

Reply:
> We can make a very readable UI by allowing users to select their controller type and then see the controls available for mapping. Ping me if you have questions on this feedback.

We already do this. I don't understand how this would change anything in the UI

Reply:
I am not at a system that i can try this out. Are you saying that I would see "WMR Left Controller Thumbstick Horizontal Axis" instead of AXIS_1 in the UI?

Reply:
No, you should def only see AXIS_1 only.

Reply:
There's no point in defining the same axis with a different name, it's redundant.

Reply:
> We can make a very readable UI by allowing users to select their controller type and then see the controls available for mapping

Again, no, far to complicated. These are to support the MRTK default configuration. We are already going to supply a custom option for users to build their own, so changing this works not help and is overly complex.

They either use what we give them (most users) and advanced users will either use the custom option, or more likely create their own controller to our spec